src/app/pages/wishlist/component/wishlist-item/wishlist-item.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="flex flex-col md:flex-row md:justify-evenly items-center md:w-128 md:pl-4">
+  <img class="rounded-md h-64 ml-0" [src]="favourite.images[0]" [alt]="favourite.name">
+
+  <div class="flex w-64 flex-col items-center md:items-start justify-center gap-5 md:pl-4">
+    <span class="text-size-24 font-bold text-blue-600">{{ favourite.name }}</span>
+
+    <span class="text-size-18 text-grey-700">{{ favourite.brandName }}</span>
+  </div>
+</div>
+
+<span class="class=w-64 text-size-24 text-center font-bold before:content-['$']">{{ favourite.price }}</span>
+
+<div class="flex items-center gap-x-4">
+  <app-general-add-to-cart-icon
+    [isExpanded]="false"
+    [productId]="favourite.id">
+  </app-general-add-to-cart-icon>
```

Comment:
Why in the `wishlist-item` component we placed some cart icon inside?
The component should be renamed